packages/derived-wallet-solana/src/shared.ts

```diff
@@ -2,7 +2,7 @@ import { makeUserApproval, makeUserRejection } from '@aptos-labs/derived-wallet-
 import { UserResponse } from '@aptos-labs/wallet-standard';
 import { WalletError } from '@solana/wallet-adapter-base';
 
-export const defaultAuthenticationFunction = '0x7::solana::authenticate';
+export const defaultAuthenticationFunction = '0x7::daa_siws_phantom::authenticate';
```

Comment:
Afaik this is not specific to Phantom right? Why is that in the name

Reply:
It is specific to phantom. From our testing wallets can generate the message in different formats, for example Nightly and Metamask on EVM. For Solana, Phantom is the only wallet that actually supports `signInWith` and when other wallets implement it we can't be sure how they will generate the message